src/worker/worker.js

```diff
@@ -15,11 +15,11 @@
 // License along with this library; if not, write to the Free Software
 // Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
 
-/* globals self: true */
+/* eslint-disable no-restricted-globals */
 
 self.window = {}; // to make UMD bundles work
 
-importScripts('openpgp.js');
+importScripts('openpgp_browser.js'); // FIXME
```

Comment:
What is this?

Reply:
See line 90 of `Gruntfile.js`. In particular, certain external parts (compare line 53 with 98) are required for running on Node but would break testing on browsers altogether, which is why I had to separate the builds. `test/unittests.html` loads `openpgp_browser.js` instead.